api/controllers/CommentController.js

```diff
@@ -172,5 +173,45 @@ module.exports = {
       })
       .then(res.ok, res.serverError)
     })
+  },
+
+  createBatchFromEmailForm: function (req, res) {
+    try {
+      var tokenData = Email.decodeFormToken(req.param('token'))
+    } catch (e) {
+      return Promise.resolve(res.serverError(new Error('Invalid token: ' + req.param('token'))))
+    }
+
+    const { communityId, userId } = tokenData
+
+    const replyText = postId => req.param(`post-${postId}`)
+
+    const postIds = flow(
```

Comment:
this is confusing me, partly because i'm used to seeing `filter((v, k) => ...)`. if all you want is the post ids, i would do: 
```js
flow(
  Object.keys,
  filter(k => k.match(/^post-(\d)+$/)),
  map(k => k.replace(/^post-/, ''))
)(req.allParams())